.gitignore

```diff
@@ -36,6 +38,7 @@ egs/*/*/tensorboard
 egs/*/*/wav*
 
 # tools related
+tools/ideep*
```

Comment:
Do you need this?

Reply:
I was trying to speed up chainer for cpu so I installed it directly, i can remove it